src/pages/docs/push/index.mdx

```diff
@@ -17,12 +17,20 @@ redirect_from:
 
 Push notifications notify user devices or browsers regardless of whether an application is open and running. They deliver information, such as app updates, social media alerts, or promotional offers, directly to the user's screen. Ably sends push notifications to devices using [Firebase Cloud Messaging](https://firebase.google.com/docs/cloud-messaging) or [Apple Push Notification Service](https://developer.apple.com/notifications/), and to browsers using [Web Push](https://developer.mozilla.org/en-US/docs/Web/API/Push_API). Push notifications don't require a device or browser to stay connected to Ably. Instead, a device's or browser's operating system or web browser maintains its own battery-efficient transport to receive notifications.
 
+<Aside data-type='see-evidence'>
+Trust 99.999% uptime with just 5 minutes of downtime per year, ensuring your realtime features are always available. [Click here for evidence 🔬](https://ably.com/docs/platform/architecture/infrastructure-operations.mdx)
```

Comment:
The SLA doesn't imply how much downtime there will be. If defines the threshold for the amount of downtime that triggers compensation. As written, this makes it sound like we're ok with 5 minutes, and customers should expect that.

Reply:
I will remove this. We only cover this in "at a glance".

Note
I'd like to expand on...  the [4 pillars](https://ably.com/four-pillars-of-dependability) in the docs. Possibly in the /platform/architecture
Leading on from the UPS work

This would have supported this metric better (in addition to the tweak in writing).